src/components/VegaGraph.tsx

```diff
@@ -0,0 +1,448 @@
+import React, { useState } from "react"
+import { View, parse } from "vega";
+import { makeStyles } from '@material-ui/core/styles';
+import Grid, { GridSpacing } from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import InputLabel from '@material-ui/core/InputLabel';
```

Comment:
Let's convert all of these imports to the same one big import in the lines below it.